### DISCUSSION TITLE
Disable Ranges

### DISCUSSION THREAD

**Discussion by SupianIDz** [malzariey/filament-daterangepicker-filter#42] [Q&A] [answered]
How to disable this ?
<img width="164" alt="Screenshot 2023-09-14 at 23 38 03" src="https://github.com/malzariey/filament-daterangepicker-filter/assets/37969970/0e52151d-a641-468a-a95a-787a530fd9dc">

**Comment by juliangums** [comment]
@SupianIDz I think `->ranges([])`

**Comment by malzariey** [comment, marked as answer]
Use the following in v2.5

```php 
->disableRanges()
```
